src/torchpme/tuning/ewald.py

```diff
@@ -174,8 +174,8 @@ def err_kspace(
         return (
             self.prefac**0.5
             / smearing
-            / torch.sqrt((2 * torch.pi) ** 2 * self.volume / (lr_wavelength) ** 0.5)
-            * torch.exp(-((2 * torch.pi) ** 2) * smearing**2 / (lr_wavelength))
+            / torch.sqrt(2 * torch.pi**2 * self.volume / lr_wavelength)
+            * torch.exp(-2 * torch.pi**2 * smearing**2 / lr_wavelength**2)
```

Comment:
Are you sure this change is correct? I think units still don't match because `smearing` and the `lr_wavelength` should cancel. I just re-derived the equation and I got

$$
ΔF_\mathrm{LR} = \frac{Q^2}{\sqrt{N}} \frac{h}{π \sqrt{V} \sigma } \mathrm{exp} \left( \frac{2 π \sigma^2}{h^2}\right)
$$

maybe check it again.

I think we should also give the conversion between our convention and the other usual one somewhere in the docs:

$$
α = \frac{1}{\sqrt{2}Σ}
K = \frac{2π}{h}
$$

where α is the inverse width of Gaussian charge cloud and $K$ is the Fourier space cutoff in inverse units of length.